modules/api/api-xsjs/src/main/java/com/sap/xsk/api/TraceFacade.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.api;
+import org.eclipse.dirigible.commons.api.scripting.IScriptingFacade;
+import org.eclipse.dirigible.commons.config.Configuration;
+
+public class TraceFacade implements IScriptingFacade{
+  private static final String XSK_DEBUG_ENABLED = "XSK_DEBUG_ENABLED";
```

Comment:
add _LOG_ prefix  -> XSK_LOG_ENABLE_DEBUG

Reply:
- `XSK_LOG_ENABLE_DEBUG` or `XSK_LOG_DEBUG_ENABLED`
- `XSK_LOG_ENABLE_ERROR` or `XSK_LOG_ERROR_ENABLED`
- `XSK_LOG_ENABLE_FATAL` or `XSK_LOG_FATAL_ENABLED`
- `XSK_LOG_ENABLE_INFO` or `XSK_LOG_INFO_ENABLED`
- `XSK_LOG_ENABLE_WARNING` or `XSK_LOG_WARNING_ENABLED`

Personally I would prefer the `XSK_LOG_XXX_ENABLED` pattern, as it will be also aligned with the current pattern in `DIRIGIBLE_XXX` related environment variables: https://www.dirigible.io/help/setup/setup-environment-variables/

Reply:
Changed with 3c4dc73. I went with @ThuF 's recommendation.